commit 61810096de3c3ec977c71dbb7e00447d70045163
Author: Borislav Petkov <bp@suse.de>
Date:   Tue Dec 15 09:18:44 2020 +0100

    EDAC/amd64: Merge error injection sysfs facilities
    
    Merge them into the main driver and put them inside an EDAC_DEBUG
    ifdeffery to simplify the driver and have all debugging/injection stuff
    behind a debug build-time switch.
    
    No functional changes.
    
    Signed-off-by: Borislav Petkov <bp@suse.de>
    Reviewed-by: Yazen Ghannam <yazen.ghannam@amd.com>
    Link: https://lkml.kernel.org/r/20201215110517.5215-2-bp@alien8.de

diff --git a/drivers/edac/amd64_edac.h b/drivers/edac/amd64_edac.h
index c072ccd3e7e2..85aa820bc165 100644
--- a/drivers/edac/amd64_edac.h
+++ b/drivers/edac/amd64_edac.h
@@ -462,10 +462,6 @@ struct ecc_settings {
 	} flags;
 };
 
-#ifdef CONFIG_EDAC_AMD64_ERROR_INJECTION
-extern const struct attribute_group amd64_edac_inj_group;
-#endif
-
 /*
  * Each of the PCI Device IDs types have their own set of hardware accessor
  * functions and per device encoding/decoding logic.